lib/live_cluster/manage_controller.py

```diff
@@ -380,11 +380,19 @@ async def _do_default(self, line):
 )
 class ManageACLChangePasswordUserController(ManageLeafCommandController):
     def __init__(self):
-        self.modifiers = set(["old", "new"])
+        self.modifiers = set(["user", "old", "new"])
         self.required_modifiers = set([])
         self.controller_map = {}
 
     async def _do_default(self, line):
+        username = util.get_arg_and_delete_from_mods(
+            line=line,
+            arg="user",
+            return_type=str,
+            default=None,
+            modifiers=self.modifiers,
+            mods=self.mods,
+        )
```

Comment:
nit: I think it's worth leaving a comment here that says this is for backwards compatibility and so the "user" modifier is not documented.